website/src/pages/privacy/cookie-policy.tsx

```diff
@@ -0,0 +1,46 @@
+import ArrowRightIcon from 'mdi-react/ArrowRightIcon'
+import * as React from 'react'
+import { Helmet } from 'react-helmet'
+import Layout from '../../components/Layout'
+export default class CookiesPage extends React.Component<any, any> {
+    constructor(props: any) {
+        super(props)
+        this.state = {
+            cookies: [],
+        }
+    }
+
+    public componentDidMount(): void {
+        if (document) {
+            document.getElementsByTagName('body')[0].setAttribute('style', 'background-image:none')
+        }
+    }
+
+    public render(): JSX.Element | null {
+        const desc =
+            'Sourcegraph cookie policy'
+        return (
+            <Layout location={this.props.location}
+                meta={{
+                    title:
+                        'Sourcegraph Cookie Policy',
```

Comment:
```suggestion
                        'Sourcegraph cookie policy',
```

Prefer sentence case